cloudbuild.yaml

```diff
@@ -87,8 +87,4 @@ steps:
   - '-u'
   - '$_HNC_USER:$_HNC_PERSONAL_ACCESS_TOKEN'
   - 'https://uploads.github.com/repos/$_HNC_REPO_OWNER/hierarchical-namespaces/releases/$_HNC_RELEASE_ID/assets?name=krew-kubectl-hns.yaml'
-# Build Docker image
-- name: gcr.io/cloud-builders/docker
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/$_HNC_IMG_NAME:$_HNC_IMG_TAG', 'hierarchical-namespaces']
 
-images: ['gcr.io/$PROJECT_ID/$_HNC_IMG_NAME:$_HNC_IMG_TAG']
```

Comment:
I'd rather keep this in for now, and have the GH Actions flow create images that are labelled differently. I don't like the idea of having two completely different flows where _both_ are necessary. I'd rather get the GH Action flow to the point where we can deprecate the Cloud Build one completely, but until then, I'd like Cloud Build to continue to be "complete."